server/v2/cometbft/server.go

```diff
@@ -72,6 +73,7 @@ func New[T transaction.Tx](
 	app appmanager.AppManager[T],
 	appCodec codec.Codec,
 	txCodec transaction.Codec[T],
+	consensusAddressCodec addresscodec.Codec,
```

Comment:
Where is this argument set?

Reply:
It should be set in the consensus struct in this function.

Reply:
yeah my bad